.github/workflows/codeql.yml

```diff
@@ -0,0 +1,78 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
```

Comment:
we use semgrep and bandit, its not clear if there's any additional benefit to adding codeql, when we were adding static analyzers, codeql at the time was quite slow.

Reply:
Understood. Thanks for clarifying

Reply:
there both at the bottom of the makefile re targets `analyzer-semgrep` and `analyzer-bandit`